spec/cell_spec.rb

```diff
@@ -48,28 +46,13 @@
   end
   
   describe "#fired_upon?" do
-    it 'will return state of fired_upon attribute' do
+    it 'has fired_upon? assigned false by default' do
       expect(@cell.fired_upon?).to eq(false)
     end
   end
 
   describe "#fire_upon" do
-    it 'will update the status of fired_upon attribute' do
-      expect(@cell.fired_upon).to eq(false)
-      @cell.fire_upon
-
-      expect(@cell.fired_upon).to eq(true)
-    end
-    
-    it 'will NOT revert status of fired_upon from true to false' do
```

Comment:
I think this block is important to keep in, the `#fire_upon` method only goes one way, It will never update the status from being `true` to `false` I think this demonstrated a thorough testing process even if the actual method works as we expect regardless of the test